frontend/src/components/Chat/GroupChat.js

```diff
@@ -19,7 +19,6 @@ const GroupChat = (props) => {
 
     useEffect(() => {
         socket.on("me", (id) => {
```

Comment:
delete this function

```suggestion
        socket.on("me", (id) => {
```